portal-impl/src/com/liferay/portlet/usersadmin/search/OrganizationMembersSearcher.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portlet.usersadmin.search;
+
+import com.liferay.portal.kernel.model.Organization;
+import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.search.BaseSearcher;
+import com.liferay.portal.kernel.search.Field;
+import com.liferay.portal.kernel.search.Indexer;
+
+/**
+ * @author Pei-Jung Lan
+ */
+public class OrganizationMembersSearcher extends BaseSearcher {
```

Comment:
This should be renamed to OrganizationUsersSearcher.

I think we should be strict to only use the word Member for associations of users to Sites or similar. The relationship of an user with an organization is much stronger than that of a membership (since the org admin can even delete the user account), so we should try to avoid the word member completely for it.